avro/pom.xml

```diff
@@ -54,7 +54,7 @@ abstractions.
     <dependency>
       <groupId>ch.qos.logback</groupId>
       <artifactId>logback-classic</artifactId>
-      <version>1.2.11</version>
```

Comment:
Ideally shouldn't add unrelated changes, but since it's test dependency, this one is fine.